CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ Documentation for rocSPARSE is available at
 
 * Add `rocsparse_create_extract_descr`, `rocsparse_destroy_extract_descr`, `rocsparse_extract_buffer_size`, `rocsparse_extract_nnz`, and `rocsparse_extract` APIs to allow extraction of the upper or lower part of sparse CSR or CSC matrices.
 * Support for the gfx1151, gfx1200, and gfx1201 architectures.
+* Added `azurelinux` OS name for correcting gfortran dependency
```

Comment:
Should this go into the Resolved issues section if it's correcting something?

Reply:
Guess I cannot answer that. I thought supporting azurelinux was new. But I can move it over, if required?

Reply:
If support for Azure Linux is new, then it can stay where it is, even if a correction is required somewhere else. Thanks for the clarification.

Reply:
I'll double check whether this is something new or not.